skrf/media/rectangularWaveguide.py

```diff
@@ -121,9 +121,10 @@ def __repr__(self):
 
     
     @classmethod
-    def from_Z0(cls,frequency, Z0,f, m=1,n=0,ep_r=1, mu_r=1, **kw):
+    def from_Z0(cls,frequency, Z0,f, ep_r=1, mu_r=1, **kw):
```

Comment:
why removing the mode index (m, n) choice ?

Reply:
That's what I mentioned in the main comment. I might be wrong, but when I derived the formula given here, the cutoff frequency is used and here, only $omega_c = c * \pi/a$ is considered, although the cutoff frequency for a non fundamental mode with different `m` and `n` than (1,0) would be $omega_c = c * \sqrt( (m \pi/a)^2 + (n \pi/a)^2)  $. Or am I wrong in the derivation?

Reply:
Sorry for the latex-like math

Reply:
You're right, I've updated my comment after reading more carefully your initial PR message. 

The formulas are (pay attention to the b-dependance of the n index)
![image](https://user-images.githubusercontent.com/4642848/88519019-ba6d1380-cff1-11ea-9f6d-d0aee3ea76a8.png)

Reply:
True! The missing b-dependence was just a typo. I have to admit that I didn't directly generalized the given calculation to any mode since the formula was a little cumbersome. But since I got it now it should be fine to implement it.
One question regarding the original formula in `rectangularWaveguide`: Is there a specific reason to have the `j` in the square of the impedance? To me it just adds complexity as it is mathematically equal to a minus sign.
So originally:
```py
1./sqrt((1/(z0*1j)**2+ep/mu))
```
could be replaced with
```py
1./sqrt((ep/mu  - 1/z0**2))
```

Reply:
Yes indeed. (Unless side effects with inverse of negative imaginary part of z0.)

However, upon reflection, initializing from z0 only works for TE10 (or TEm0, or TMm0), but not for TEmn or TMmn. In the latter case, both `a` and `b` are unknown and can't be retrieved from z0 only.